hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/RaftBackendStoreProvider.java

```diff
@@ -119,6 +120,17 @@ public synchronized BackendStore loadGraphStore(String name) {
         return this.graphStore;
     }
 
+    @Override
+    public synchronized BackendStore loadGraphStore(HugeConfig config,String name) {
```

Comment:
we can just update loadGraphStore at line 112, and also update loadSchemaStore and loadSystemStore to keep the same style